pkg/chartverifier/profiles/profile_test.go

```diff
@@ -0,0 +1,121 @@
+package profiles
+
+import (
+	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/checks"
+	"github.com/stretchr/testify/assert"
+	"path/filepath"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+func TestProfile(t *testing.T) {
+
+	testProfile := Profile{}
+
+	testProfile.Apiversion = "v1"
+	testProfile.Kind = "verifier-profile"
+	testProfile.Name = "profile-1.0.0"
+
+	testProfile.Annotations = []Annotation{DigestAnnotation, OCPVersionAnnotation, LastCertifiedTimestampAnnotation}
+
+	testProfile.Checks = []*Check{
```

Comment:
is this not the same as the result of `getDefaultProfile`? could not we call that function instead here?

Reply:
Done, also it highlighted a mistake in the defaultProfile :-)